tests/conftest.py

```diff
@@ -266,21 +266,10 @@ def algolia_exception(*args):
                  side_effect=algolia_exception)
 
 
-@pytest.fixture(scope='function')
-def fake_validation(mocker):
```

Comment:
I think we were going to use this at some point for something. But since we're not 🤷‍♂